docs/docs/data-stores/pre-built-actions/README.md

```diff
@@ -0,0 +1,74 @@
+# Using pre-built Data Store actions
+
+Pipedream includes several native pre-built actions to interact with your Data Stores for the most common operations to speed up your workflow development.
```

Comment:
remove "native". Generally I refer to them as "pre-built" only in the docs, which I think communicates the "native" point too

Reply:
I think you can remove "to speed up your workflow development" and the sentence will still communicate your point clearly.